src/test/ui/rfc-2457/no_mangle_nonascii_forbidden.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0754]: `#[no_mangle]` requires ASCII identifier
+  --> $DIR/no_mangle_nonascii_forbidden.rs:3:1
+   |
+LL | #[no_mangle]
+   | ^^^^^^^^^^^^
```

Comment:
The span doesn't cover the non-ASCII identifier.
I'd rather use `guess_head_span` here.

Reply:
Ok, will do.